packages/widgets/Source/BaseLayerPicker/BaseLayerPickerViewModel.js

```diff
@@ -271,17 +281,21 @@ function BaseLayerPickerViewModel(options) {
     options.selectedImageryProviderViewModel,
     imageryProviderViewModels[0]
   );
-  this.selectedTerrain = defaultValue(
-    options.selectedTerrainProviderViewModel,
-    terrainProviderViewModels[0]
-  );
+
+  Promise.resolve(
```

Comment:
Similar to my other comment, if this code is permanent, it's still problematic because you potentially have unhandled promise rejections.